x-pack/plugins/index_management/__jest__/client_integration/index_template_wizard/template_clone.helpers.ts

```diff
@@ -21,4 +21,4 @@ const testBedConfig: TestBedConfig = {
 
 const initTestBed = registerTestBed(WithAppDependencies(TemplateClone), testBedConfig);
 
-export const setup = formSetup.bind(null, initTestBed);
+export const setup: any = formSetup.bind(null, initTestBed);
```

Comment:
Just curious, why are these needed?

Reply:
The type isn't inferred from `formSetup` and the typescript project couldn't build.

Reply:
Ah OK. It is surprising that `.bind` returns `any`! What we'd like to express is:

```ts
const a = (a: number) => {};

const b: () => ReturnType<typeof a> = a.bind(null, 1);
```

Not sure what else could be done, other than enabling `strictBindCallApply` in tsconfig, which is a breaking type check change -- but would solve this for us :)